librz/include/rz_il/rzil_opcodes.h

```diff
@@ -275,7 +275,6 @@ struct rzil_op_bool_operation_t {
  */
 struct rzil_op_bool_inv_t {
 	RzILOp *x; ///< index of the BOOL operand
-	RzILOp *ret; ///< index of store the BOOL result
```

Comment:
Why removing this? RZIL is pure and should not change operands in place

Reply:
I think that is exactly the point of this change, the `ret` looks like an old leftover to me.

Reply:
it's a leftover it is not used and there is no reason for being there because inverting a number requires only 1 argument.